frontend/src/components/Chat/ChatContainer.tsx

```diff
@@ -66,19 +66,27 @@ export default function ChatContainer({
       </div>
     );
   }
+
+  const chatInput = <div className='chat-input'>
+    <ChatInput
+      messageType={chainType}
+      directMessageId={directMessageID}
+      isDisabled={!messageChain.isActive}

```

Comment:
I don't think you need the isDisabled prop anymore, since we aren't rendering the ChatInput box when the chain is disabled. Do you mind ripping that out?

Reply:
oh yes! now only the canSendMessage within ChatInput matters